### DISCUSSION TITLE
How to use project.nvim

### DISCUSSION THREAD

**Discussion by rubiin** [LazyVim/LazyVim#2473] [Q&A] [answered]
Currently,  the section talks about project browsing `https://www.lazyvim.org/extras/util/project` . However I cannot seem to find how we are creating project . The readme section of `project.nvim` talks about how it searches for projects but when I type the keybind, it always shows empty

**Comment by dpetka2001** [comment]
You have to manually call `:ProjectRoot`, because LazyVim by default sets `manual = true`. If you set it to `false` in your personal configuration it will automatically detect I believe. Not 100% sure as I don't use the plugin.

**Reply by rubiin** [reply]
let me check

**Reply by rubiin** [reply]
Works. But  I am pretty sure the readme says this is just for changing the root to cwd. By default , the plugin should add all the dirs where git is initialized. Must be a plugin issue .

**Reply by dpetka2001** [reply]
It was changed to `manual = true`, because there were some issues where the plugin changed the LSP root directory and caused some problems when searching for files. So, it was changed to `manual = true` and if a user wanted he could change it in his own personal configuration, assuming he knows what he's doing. I'm not using the plugin myself as i prefer [telescope-project](https://github.com/nvim-telescope/telescope-project.nvim/) for projects, so I don't know what to tell you. 

Also I'm just a regular user like you and don't make any decisions. I'm just telling you what the reason behind this change was. If you want, you can change it in your personal configuration and use it as you see fit.

**Reply by rubiin** [reply]
https://github.com/ahmedkhalf/project.nvim/issues/61 something similar, did some digging

**Reply by rubiin** [reply, marked as answer]
with manual_mode=true, the correct way of doing is `AddProject` autocmd

**Reply by liangmiQwQ** [reply]
And we how to remove a project in the list?

**Reply by vsuharnikov** [reply]
If you select the project in `SPC f p` and press `Ctrl d`, then a dialog appears. But it doesn't work for me, I see an error when trying to delete a project:

> E5108: Error executing lua: ...vim/lazy/project.nvim/lua/project_nvim/utils/history.lua:63: bad argument #1 to 'ipairs' (table expected, got nil)

UP: I was able to delete a project from the dashboard
